### PR TITLE
Add Over to List

### PR DESCRIPTION
**Pull request by bertadevant**
This is a modified version of the [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/master/CONTRIBUTING.md).

This pull request adheres to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/blob/master/CODE_OF_CONDUCT.md).

- [x] I am an employee of the company mentioned and confirm all included details are correct
- [x ] You know your alphabet - company is listed in alphabetical order in the README
- [x] The company directly hires employees. No bootcamps / freelance sites / etc
- [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such
- [x] A [company profile](https://github.com/remoteintech/remote-jobs/blob/master/company-profiles/example.md) is included - __Required__ for new additions. (This can be a basic outline but at least something please)
- [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [x] __Region__ details the geographic regions in which this company's employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

**Comment by dougaitken** [comment]
Hey @bertadevant, thanks for this! I met the co-founder of Over on a flight at the weekend. Such timing for your PR, I appreciate that!

File: company-profiles/over.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+Worldwide
+
+### Example region 1
+USA
+
+### Example region 2
+North America, Latin America, Caribbean
+
+### Example region 3
+Worldwide
```
Comment (nylen) [inline company-profiles/over.md]: Let's remove the "Example region" stuff from this profile... @bertadevant @dougaitken can you help with this?

Comment (bertadevant) [reply company-profiles/over.md]: Yeah I will tackle it in the morning!

Comment (nylen) [reply company-profiles/over.md]: See #552.

**Pull request merged**